init.ts

```diff
@@ -1,24 +1,48 @@
 import { ensureFile, join, resolve } from "./src/deps.ts";
 
-const routesDirectory = resolve("./routes");
+export async function createSitemap(url: string | null) {
+  const routesDirectory = resolve("./routes");
 
-const destination = join(routesDirectory, "sitemap.xml.ts");
+  const destination = join(routesDirectory, "sitemap.xml.ts");
 
-await ensureFile(destination);
+  await ensureFile(destination);
 
-const stub = `import { Handlers } from "$fresh/server.ts";
+  const stub = `import { Handlers } from "$fresh/server.ts";
 import manifest from "../fresh.gen.ts";
 import { SitemapContext } from "https://deno.land/x/fresh_seo@0.1.1/mod.ts";
-
 export const handler: Handlers = {
   GET(req, ctx) {
-    const sitemap = new SitemapContext("http://example.com", manifest);
-
+    const sitemap = new SitemapContext("${
+    url ?? "http://exmaple.com"
+  }", manifest);
     // You can add additional page here
-
     return sitemap.render();
   },
 };
 `;
 
-await Deno.writeTextFile(destination, stub);
+  return Deno.writeTextFile(destination, stub);
+}
+
+export async function createRobotTxt(url: string | null) {
+  const directory = resolve("./static");
```

Comment:
Fresh introduced customization of static folder location. Maybe this function should take a parameter to specify? Or even better yet, this function should create a `routes` entry and return the robots.txt content

Reference: denoland/fresh#626

Reply:
Please offer me one clarification:
What did you mean with?
> function should create a `routes` entry

Reply:
I think what @notangelmario meant was that we could support the same pattern as for `sitemap.xml` which create a route (`routes/sitemap.xml.ts`) to support generating the file on-demand.

I think it's not mandatory in this PR though, as `robot.txt` are not subject to change as much as `sitemap.xml` so we could keep it like that and add it later if needed

Reply:
I guess that's right. Fresh will soon support plugins that will let us create routes ourselves and we might not even need an `init.ts` after all